### PR TITLE
Get association target name from target object

### PR DESCRIPTION
**Pull request by p-fedyukovich**
Related to #34

### CODE REVIEW

**Comment by DanielHreben** [comment]
Is that enough to close #34 ?

**Comment by p-fedyukovich** [comment]
I think so, but may be need to check many-to-many association as well

**Comment by DanielHreben** [comment]
This looks good to me, and I don't mind to merge. It will be really nice to add some tests covering those cases. I will do it eventually but don't have enough time for that now. 
@ignacioblit, @fraywing, how does it looks for you guys?

File: packages/sequelize-transparent-cache/src/cache/util.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ function loadAssociations (model) {
   Object.keys(model.associations).forEach((key) => {
     //  model.associations[key] does not work on include, we grab it from sequelize.model()
     if (model.associations[key].hasOwnProperty('options')) {
-      const modelName = model.associations[key].options.name.singular
+      const modelName = model.associations[key].target.name
       associations.push({
         model: model.sequelize.model(modelName),
```
Comment (ignacioblit) [inline packages/sequelize-transparent-cache/src/cache/util.js]: Even though this way of fetching the name is more 'secure', not having a check if the model is defined would risk throwing an unhandled error just like described on https://github.com/DanielHreben/sequelize-transparent-cache/issues/34

**Pull request merged**